rust/lancedb/src/connection.rs

```diff
@@ -691,13 +699,49 @@ mod tests {
         // let db = Database::connect("s3://bucket/path/to/database").await.unwrap();
     }
 
+    #[tokio::test]
+    #[ignore = "this can't pass due to https://github.com/lancedb/lancedb/issues/1019, enable it after the bug fixed"]
+    async fn test_open_table() {
+        let tmp_dir = tempdir().unwrap();
+        let uri = tmp_dir.path().to_str().unwrap();
+        let db = connect(uri).execute().await.unwrap();
+
+        assert_eq!(db.table_names().await.unwrap().len(), 0);
+
+        db.open_table("invalid_table").execute().await.unwrap();
```

Comment:
Won't this `unwrap` cause a panic since the table doesn't exist?  Should this line be removed?

Reply:
you are right, will fix this in the enabling PR